qiskit/visualization/circuit/latex.py

```diff
@@ -47,7 +48,24 @@ class QCircuitImage:
     Thanks to Eric Sabo for the initial implementation for Qiskit.
     """
 
-    def __init__(
+    @deprecate_arg("gregs", since="0.20.0")
+    @deprecate_arg("cregs", since="0.20.0")
+    @deprecate_arg("layout", since="0.20.0")
+    @deprecate_arg("global_phase", since="0.20.0")
+    @deprecate_arg(
+        "circuit",
+        deprecation_description=(
+            "Not setting the ``circuit`` argument in the ``QCircuitImage`` constructor"
+        ),
+        since="0.20.0",
+        predicate=lambda circuit: circuit is None,
+        additional_msg=(
+            "The ``circuit`` argument must be a valid QuantumCircuit. If ``circuit`` is set to "
+            "``None``, then the circuit will be built using the ``qubits`` and ``clbits`` "
+            "arguments for rendering the drawing."
+        ),
+    )
```

Comment:
imo: "not setting x is deprecated" is a double negative, except the second is hiding in the jargon word "deprecated". Then the next sentence says "it will be removed"; _what_ will be removed?  That's "'not setting the circuit argument' will be removed", but "not setting the circuit argument" is a user behaviour, and not something we can remove.  The original form is crystal clear on everything except the timeline for removal.

I don't know if my point is clear, sorry - I just mean that I already find that wording quite verbose and still confusing, and I easily understand what "deprecate" means, which not all our users do.